src/backends/perf_pt/perf_pt.c

```diff
@@ -346,14 +346,18 @@ perf_pt_init_tracer(struct tracer_conf *tr_conf)
 
     // Allocate mmap(2) buffers for speaking to perf.
     //
-    // These regions are mmap(2)ed from the perf fd in two separate chunks:
+    // We mmap(2) two separate regions from the perf file descriptor into our
+    // address space:
     //
-    //            "Base buffer"
-    // -----------------------------------     ------------------
-    // | header  |       data buffer     | ... |   AUX buffer   |
-    // -----------------------------------     ------------------
+    // 1) The base buffer (tr_ctx->base_buf), which looks like this:
+    //
+    // -----------------------------------
+    // | header  |       data buffer     |
+    // -----------------------------------
     //           ^ header->data_offset
     //
+    // 2) The AUX buffer (tr_ctx->aux_buf), which is a simple array of bytes.
```

Comment:
I don't think "which is a simple array of bytes" adds anything useful, and can be deleted.